tests/wallet/vc_wallet/test_vc_wallet.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import annotations
+
+from typing import Any, Optional
+
+import pytest
+
+from chia.rpc.wallet_rpc_client import WalletRpcClient
+from chia.simulator.full_node_simulator import FullNodeSimulator
+from chia.simulator.time_out_assert import time_out_assert_not_none
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.peer_info import PeerInfo
+from chia.util.ints import uint16, uint64
+from chia.wallet.did_wallet.did_wallet import DIDWallet
+from chia.wallet.util.wallet_types import WalletType
+from chia.wallet.vc_wallet.vc_store import VCProofs, VCRecord
+
```

Comment:
we need negative tests, here are some:
 - try to mint a VC with wrong DID 
 - try to revoke a VC with wrong DID 
 - try to update proofs on a VC after it was transferred to the user 
 - try to prove proofs that are not in the VC 
 - try to update proofs without having a valid DID 
 - try to spend a VC's backdoor with invalid DID 
 - try to prove a tx with someone else's VC

Reply:
I believe I have addressed all of these now (I implemented most of the new ones in the chialisp lifecycle tests because it is more flexible with passing in wrong values etc.)

The exception is "try to update proofs on a VC after it was transferred to the user" because this is not something we support and I would expect such a test to fail.